lib/components.dart

```diff
@@ -0,0 +1,21 @@
+export 'src/components/component.dart';
+export 'src/components/base_component.dart';
+export 'src/components/isometric_tile_map_component.dart';
+export 'src/components/nine_tile_box_component.dart';
+export 'src/components/parallax_component.dart';
+export 'src/components/particle_component.dart';
+export 'src/components/position_component.dart';
+export 'src/components/sprite_animation_component.dart';
+export 'src/components/sprite_batch_component.dart';
+export 'src/components/sprite_component.dart';
+export 'src/components/text_box_component.dart';
+export 'src/components/text_component.dart';
+
+export 'src/components/mixins/draggable.dart';
+export 'src/components/mixins/has_game_ref.dart';
+export 'src/components/mixins/single_child_particle.dart';
+export 'src/components/mixins/tapable.dart';
+
+export 'src/extensions/vector2.dart';
```

Comment:
should we make a fail extensions.dart on the extensions folder to export all our extensions? and then export that file everywhere instead of only vector2 ext